components/compositing/compositor.rs

```diff
@@ -96,17 +96,50 @@ impl FrameTreeId {
     }
 }
 
+/// Data that is shared by all WebView renderers.
+pub struct ServoRenderer {
+    /// Our top-level browsing contexts.
+    webviews: WebViewManager<WebView>,
+
+    /// Tracks whether we are in the process of shutting down, or have shut down and should close
+    /// the compositor.
+    shutdown_state: ShutdownState,
+
+    /// The port on which we receive messages.
+    compositor_receiver: CompositorReceiver,
+
+    /// The channel on which messages can be sent to the constellation.
+    constellation_sender: Sender<ConstellationMsg>,
+
+    /// The channel on which messages can be sent to the time profiler.
+    time_profiler_chan: profile_time::ProfilerChan,
+
+    /// The webrender interface, if enabled.
```

Comment:
This comment probably needs to be updated - is it ever not enabled? `/// The interface to talk to the WebRender backend` maybe?

Reply:
Good point. I've updated it.